eng/scripts/Deploy-TestResources.ps1

```diff
@@ -19,11 +21,42 @@ function New-StringHash([string[]]$strings) {
     return [BitConverter]::ToString($hashBytes) -replace '-', ''
 }
 
-$testResourcesDirectory = Resolve-Path -Path  "$RepoRoot/areas/$($Area.ToLower())/tests" -ErrorAction SilentlyContinue
+switch ($PSCmdlet.ParameterSetName) {
+    'ByPath' {
+        if(!(Test-Path -Path $Path)) {
```

Comment:
Should this be a tool path or can it be a root path?

Reply:
If this requires to be a path to the bicep we should probably add validation for that.

Reply:
It's not a path to bicep, but a path that should contain bicep at "{PATH}/tests/test-resources.bicep".  This is validated on line 57

Reply:
I've updated the params to be Path + wildcard based